packages/core/package-manager/src/NodePackageManager.js

```diff
@@ -71,15 +71,68 @@ export class NodePackageManager implements PackageManager {
     |},
   ): Promise<any> {
     let {resolved} = await this.resolve(name, from, opts);
-    return this.load(resolved, from);
+    return await this.load(resolved, from);
```

Comment:
The linter complains about this `await`!

Reply:
That await isn't needed. If you await, the function implicitly returns a promise. That's basically the same as returning the promise returned by `load` directly.

Reply:
Right. I am used to awaiting all the promises no matter if they are on return or not. But this works too.